src/glue/jobs/s3_to_json_s3.py

```diff
@@ -125,6 +125,7 @@ def process_record(s3_obj, s3_obj_metadata, dataset_mapping):
                 with open(output_path, "w") as f_out:
                     json.dump(j, f_out, indent=None)
                     s3_output_key = os.path.join(
+                        workflow_run_properties["namespace"],
```

Comment:
Why do we add this to the workflow run properties? It's already in the workflow name and we don't actually do anything with it in the job.

Reply:
Can you be more clear with what you want to happen here instead?

Reply:
I originally misunderstood the purpose of this line. It seems fine.